catroid/src/main/java/org/catrobat/catroid/ui/fragment/CategoryBricksFactory.java

```diff
@@ -317,6 +321,24 @@
 //		return false;
 //	}
 
+	private List<Brick> setupChromecastCategoryList(Context context) {
+		List<Brick> chromecastBrickList = new ArrayList<Brick>();
+		chromecastBrickList.add(new WhenGamepadButtonBrick(new WhenGamepadButtonScript(context.getString(R.string
+				.cast_gamepad_A))));
```

Comment:
im not a huge fan of breaking stuff like R.string.someResourceId apart into two lines, it makes it way harder to read. so why not move the whole context.getString.... to new line.